Connectors/Autocad/Speckle.Connectors.AutocadShared/Bindings/AutocadSendBaseBinding.cs

```diff
@@ -86,6 +86,10 @@ ISpeckleApplication speckleApplication
     {
       _sendConversionCache.ClearCache();
     };
+    _idleManager.SubscribeToIdle(
```

Comment:
very confused, why we need to subscribe to idle on constructor? Also isn't this PR receive fix, why we are touching send?